R/summary.R

```diff
@@ -152,6 +152,9 @@ summary.fixed_design <- function(object, ...) {
 #'   variables you want to be displayed differently than the defaults.
 #' @param col_vars The variables to be displayed.
 #' @param col_decimals The decimals to be displayed for the displayed variables in `col_vars`.
+#'   If the vector is unnamed, it must match the length of `col_vars`. If the
+#'   vector is named, you only have to specify the number of digits for the
+#'   columns you want to be displayed differently than the defaults.
```

Comment:
Hi @jdblischak , can we add an example at `@examples` to show the usage of named/unnamed vector in `summary`?